### DISCUSSION TITLE
Embedart issue - IM result is not a number

### DISCUSSION THREAD

**Discussion by undaunt** [beetbox/beets#5219] [Q&A]
Hello all,

I'm seeing some errors with the following config when trying to embed art. Any insight would be super appreciated!

Config:
```
embedart:
  auto: yes
  compare_threshold: 50
  ifempty: yes
  maxwidth: 0
  remove_art_file: no
```

Here is some log output:

```
embedart: Extracting album art from: Artist - Album - Song to: /tmp/tmpkpnxq_h7.jpg
embedart: comparing images with pipeline ['convert', b'/mnt/music/content/Artist/Album (2015)/cover.jpg', b'/tmp/tmpkpnxq_h7.jpg', '-colorspace', 'gray', 'MIFF:-'] | ['compare', '-metric', 'PHASH', '-', 'null:']
embedart: IM output is not a number: ''
embedart: Image not similar; skipping.
```

When I run a similar manual test against the docker container via direct `magick` invocation, I get a result of 0.

`docker exec -ti beets bash -c 'magick convert /mnt/music/content/Artist/Album\ \(2015\)/cover.jpg /tmp/tmpkpnxq_h7.jpg -define phash:colorspaces=sRGB,HCLp -colorspace gray MIFF:- | compare -metric PHASH - null:'`
or
`docker exec -ti beets bash -c 'magick convert /mnt/music/content/Artist/Album\ \(2015\)/cover.jpg /tmp/tmpkpnxq_h7.jpg -colorspace gray MIFF:- | compare -metric PHASH - null:'`

**Comment by wisp3rwind** [comment]
Which version of beets is this? Might be https://github.com/beetbox/beets/issues/4272

**Comment by undaunt** [comment]
Running the latest version of the LSIO docker image.

Here's my output:

```
beets -vv version
user configuration: /config/config.yaml
data directory: /config
plugin paths: 
artresizer: method is (2, (7, 1, 1), False)
Sending event: pluginload
library database: /config/library.db
library directory: /mnt/music/content
Sending event: library_opened
beets version 1.6.0
Python version 3.11.9
plugins: albumtypes, badfiles, bandcamp, chroma, convert, copyartifacts, duplicates, edit, embedart, export, fetchart, fromfilename, ftintitle, hook, info, lastgenre, lyrics, mbsubmit, mbsync, missing, originquery, parentwork, permissions, replaygain, the, unimported, web, xtractor
Sending event: cli_exit
hook: running command "echo All tasks finished!" for event cli_exit
All tasks finished!
```

**Reply by wisp3rwind** [reply]
That's definitely a development version of beets that's at least ~2 years old. Probably, this is fixed with a current snapshot. Whatever LSIO is, it's not affiliated with beets directly: You should ask them to update.
